dll/win32/shell32/wine/shell32_main.c

```diff
@@ -1158,7 +1197,16 @@ static INT_PTR CALLBACK AboutDlgProc( HWND hWnd, UINT msg, WPARAM wParam, LPARAM
                 WCHAR szAuthorsText[20];
 
                 // Preload the ROS bitmap
-                hLogoBmp = (HBITMAP)LoadImage(shell32_hInstance, MAKEINTRESOURCE(IDB_REACTOS), IMAGE_BITMAP, 0, 0, LR_DEFAULTCOLOR);
+                if (IsVersionWorkstation())
```

Comment:
Why not use `IsWindowsServer` from `versionhelpers.h`?

Reply:
Great Suggestion! That cleaned up most of the code.